### DISCUSSION TITLE
cpu/memory displayed in GCP

### DISCUSSION THREAD

**Discussion by saloni-2404** [strimzi/strimzi-kafka-operator#4273] [General]
Hi,
The cpu/memory value that gets displayed in the GCP for a service, for instance kafka external bootstrap, should ideally show the cpu/memory as an aggregate of all the kafka brokers, right?
Scenario:
20brokers are there,
The cpu/memory in GCP console keeps changing, sometime reflecting for 2 brokers or 8 etc. Is this expected behaviour?

**Comment by scholzj** [comment]
Hi @saloni-2404,

We do not have any special relationship with GCP and I do not know what or how their consoles work. I do not know how it is implemented and why it does not work. Maybe the fact that each pod can have multiple services is not something they can handle? I think if it does not work for you, you need to raise this with the GCP support. If they explain what the problem is, we can think if it would be possible to fix it on Strimzi side or not.

Jakub

**Reply by saloni-2404** [reply]
okay sure Jakub, will try reaching out to Google support. Thanks
